fofix/game/Song.py

```diff
@@ -2104,23 +2108,31 @@ def __init__(self, engine, infoFileName, songTrackName, guitarTrackName, rhythmT
         except Exception, e:
             Log.warn("Unable to load rhythm track: %s" % e)
 
-
+        i = 3
         try:
-            if drumTrackName:
-                self.drumTrack = Audio.StreamingSound(self.engine.audio.getChannel(3), drumTrackName)
+            if drumTrackNames:
+                for drumTrackName in drumTrackNames:
```

Comment:
In this loop instead of what you have you could use enumerate()

for i, drumTrackName in enumerate(drumTrackNames):